openage/util/fslike/test.py

```diff
@@ -181,8 +191,9 @@ def test():
     # test the union
     test_union(root_path, root_dir)
 
-    # test the case ignoring dir
-    test_case_ignoring(root_path, root_dir)
+    if is_filesystem_case_sensitive():
+        # test the case ignoring dir
+        test_case_ignoring(root_path, root_dir)
```

Comment:
We should not skip it completely, but only the part where the assumption of a non-insensitive fs is done. I already addressed this in #895. Your `is_filesystem_case_sensitive` function is better for the detection though, so I integrated it into my pull. Thanks :)

Reply:
Oops. I didn't check your PR.
I'll revert changes related to fs from mine.